setup.cfg

```diff
@@ -46,12 +46,14 @@ install_requires =
     matplotlib<3.5  # this is for nilearn, which doesn't include it in its reqs
     nibabel>=3.0.0
     nilearn>=0.7.1
+    numba
```

Comment:
is this used?

Reply:
I think it is used by sparse. I got an error in the testing when numba wasn't included.

Reply:
Can you add a comment to that effect?
```suggestion
    numba  # used by sparse
```